fs/operations/rc_test.go

```diff
@@ -306,6 +306,26 @@ func TestRcStat(t *testing.T) {
 	})
 }
 
+// operations/settier: Set the storage tier of a file
+func TestRcSettier(t *testing.T) {
+	// ctx := context.Background()
+	r, call := rcNewRun(t, "operations/settier")
+	file1 := r.WriteObject(context.Background(), "file1", "file1 contents", t1)
+	r.CheckRemoteItems(t, file1)
+
+	if r.Fremote.Features().SetTier {
+		in := rc.Params{
```

Comment:
Getting this test right is tricky because we don't know the valid values for tiers.

I think I would get an object with `o, err = f.NewObject(ctx, file1.Path)` read the tier from that by using a type assertion `o.(fs.GetTierer)` and then calling `GetTier()` to read the current tier.

I'd then set the object to its current tier.

Reply:
I believe this is doing the correct thing now, but would appreciate a second look if you could 😁